drivers/gles3/rasterizer_storage_gles3.cpp

```diff
@@ -5105,11 +5105,13 @@ void RasterizerStorageGLES3::update_dirty_multimeshes() {
 
 			glBindBuffer(GL_ARRAY_BUFFER, multimesh->buffer);
 			uint32_t buffer_size = multimesh->data.size() * sizeof(float);
-			if (config.should_orphan) {
-				glBufferData(GL_ARRAY_BUFFER, buffer_size, multimesh->data.ptr(), GL_DYNAMIC_DRAW);
-			} else {
-				glBufferSubData(GL_ARRAY_BUFFER, 0, buffer_size, multimesh->data.ptr());
-			}
+
+			// this could potentially have a project setting for API options as with 2d
```

Comment:
You mean there should be separate settings for 2D and 3D?

Side note, might be worth removing the extra tabs on the commented lines, commenting out code on column 0 doesn't work well with automated code formatting.

Reply:
I did combine them in the beta 2 build for convenience, but we haven't had issues about slowness here from the betas afaik, so I'm satisfied that fixing it to the current version should be ok for now.